dimod/generators/knapsack.py

```diff
@@ -20,7 +20,7 @@
 from dimod.constrained import ConstrainedQuadraticModel
 from dimod.typing import ArrayLike
 
-__all__ = ['knapsack', 'random_knapsack']
+__all__ = ['knapsack', 'random_knapsack', 'quadratic_knapsack']
```

Comment:
Now for a true nit, alphabetical is easier to maintain. Likewise for the order of definition below.

Reply:
Hahaha in my head it was alphabetical because q feels so late in the alphabet... I should go back to elementary school

Reply:
now it's real alphabetical not Kate's version of alphabetical 😂 https://github.com/dwavesystems/dimod/pull/1391/commits/cded3763ef23998b7294b086138501874963287d

Reply:
FWIW, I'm with you. It feels like it should be "rstquv".

Reply:
yeah that would make way more sense, thank you